dropwizard-client/src/test/resources/yaml/dropwizardApacheConnectorTest.yml

```diff
@@ -0,0 +1,5 @@
+# this is needed to start the application in the DropwizardApacheConnectorTest
+server:
+  applicationConnectors:
+    - type: http
+      port: 34480
```

Comment:
We could use zero and OS will assign a random TCP port for the server.
Doing so, we make the test more independent from environment.